Documentation/ExtensionArchitecture/BestPractises/_ext_localconf.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+use TYPO3\CMS\Core\Utility\ExtensionManagementUtility;
+
+defined('TYPO3') or die();
+
+// Add your code here
+ExtensionManagementUtility::doSomething();
```

Comment:
ExtensionManagementUtility does not have a doSomething method?

Reply:
Nope. But anyone who copies this line of code should notice that he needs to exchange this with whatever he wants to do?

If I put nothing then the use statement gets removed by the cgl and we want to demonstrate in this very code snippet where to put the use statements and where to put the code that does something.

Reply:
Sorry to bother, but see no change or reaction?